dashboard/common/utils.py

```diff
@@ -9,15 +9,13 @@
 
 
 def find_earliest_start_datetime_of_courses():
-    course_start_datetimes = []
-    for course in Course.objects.all():
-        course_start_datetimes.append(course.get_course_date_range().start)
-    logger.debug(course_start_datetimes)
-
-    earliest_start = None
-    if len(course_start_datetimes) > 0:
-        earliest_start = sorted(course_start_datetimes)[0]
-        logger.info(f"Earliest start datetime for all courses: {earliest_start.isoformat()}")
+    sorted_courses = sorted(Course.objects.all(), key=lambda course: course.course_date_range.start)
+
+    earliest_course = None
```

Comment:
I think this needs to be `earliest_start` like it was before.`

Reply:
This WOULD cause an error I think if line 138  in `cron.py` wasn't already checking whether the number of courses was zero. It actually makes this `if/else` arrangement look unnecessary, but I suppose more precautionary null handling isn't a huge deal.

Reply:
Yeah, you're right about changing the wrong variable. The if/else is mostly in support of the logging rather than the null handling I think.